imports/include/nidcpowerObsolete.h

```diff
@@ -0,0 +1,56 @@
+/****************************************************************************
+ *              National Instruments DC Power Supplies and SMUs
+ *--------------------------------------------------------------------------*
+ *      Copyright (c) National Instruments 2016.  All Rights Reserved.      *
+ *--------------------------------------------------------------------------*
+ *
+ * Title:    nidcpowerObsolete.h
```

Comment:
@ganeshnie , **nidcpower.h** uses the line **#include "nidcpowerObsolete.h"** , so we have to include this file also else we would get build errors.

Reply:
We should add the obsolete headers to the `include` directory to avoid needing to modify `nidcpower.h`, but we shouldn't add service handlers fo rthem.

Reply:
@jonlugoatx , There are only 3 Obsolete APIs, I checked they are not present as part of metadata and hence they are also generated in the proto and service handlers.